### PR TITLE
Cameron's branch

### PR DESCRIPTION
**Pull request by CIsbis**
Welcome.py adjusted, CSV locations adjusted

### CODE REVIEW

File: src/ReadCSVFile.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def fix_working_directory(self):
     def get_file_data(self,file_name):
         self.fix_working_directory()
         file_data = []
-        with open("resource/" + file_name, 'rt')as data_file:
+        with open("src/" + file_name, 'rt')as data_file:
```
Comment (ScottR14) [inline src/ReadCSVFile.py:13]: Why is this going to the source file. You TWAT

File: src/Welcome.py [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
         print("You are logged in")
-    elif user_input == "2":
-        print("You are registered")
-    elif user_input == "3":
-        print("Search for cars")
-    elif user_input == "4":
+        print("Enter 3 to search for cars")
+        print("Enter 4 to sell a car")
+        print("Enter 5 to exit program")
+        while True:
```
Comment (ScottR14) [inline src/Welcome.py]: What's the point on this "while true" can I get some clarification?

File: src/Welcome.py [hunk 3]
```diff
@@ -1,27 +1,45 @@
...
     print("Enter 2 to register")
-    print("Enter 3 to Search for cars")
-    print("Enter 4 to exit program")
+    # print("Enter 3 to Search for cars")
+    # print("Enter 4 to sell a car")
+    print("Enter 5 to exit program")
     user_input = input("Enter Number: ")
     return user_input
 def user_input(user_input):
+    if user_input == "2":
```
Comment (ScottR14) [inline src/Welcome.py]: Could we use a switch statement here

File: src/Welcome.py (hunk 2 above)
Comment (CIsbis) [reply src/Welcome.py]: Fixed

**Pull request closed without merging**